config/src/main/java/com/alibaba/nacos/config/server/service/sql/EmbeddedStorageContextUtils.java

```diff
@@ -46,12 +46,13 @@ public class EmbeddedStorageContextUtils {
      * @param sql  sql
      * @param args argument list
      */
-    public static void addSqlContext(String sql, Object... args) {
+    public static void addSqlContext(String sql, boolean necessary, Object... args) {
```

Comment:
I think creating a new overload method is a better way ,and  many caller methods do not need modify

Reply:
besides，the meaning of 'necessary' is not very clear for me, would you please explain it？
I think it is a transaction meaning, if this operation fails ,all operation should rollback, if so，can we change to a more accurate param name or add a detailed commentary  .

Reply:
@shiyiyue1102 Please review again， thank you.